moler/cmd/unix/gunzip.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""
+Gunzip command module.
+"""
+
+__author__ = 'Adrianna Pienkowska'
+__copyright__ = 'Copyright (C) 2018, Nokia'
+__email__ = 'adrianna.pienkowska@nokia.com'
+
+from moler.cmd.unix.genericunix import GenericUnixCommand
+from moler.exceptions import CommandFailure
+from moler.exceptions import ParsingDone
+import re
+
+
+class Gunzip(GenericUnixCommand):
+    def __init__(self, connection, archive_name, output_file_name=None, options=None, overwrite=False):
+        super(Gunzip, self).__init__(connection=connection)
+        self.archive_name = archive_name
+        self.output_file_name = output_file_name
+        self.options = options
+        self.overwrite = overwrite
+        self.keys = list()
+        self.current_ret['RESULT'] = list()
+
+    def build_command_string(self):
+        cmd = 'gunzip'
+        if self.options:
+            cmd = '{} {}'.format(cmd, self.options)
+        if self.archive_name:
+            for file in self.archive_name:
+                cmd = '{} {}'.format(cmd, file)
+        if self.output_file_name:
+            cmd = '{} > {}'.format(cmd, self.output_file_name)
+        return cmd
+
+    def on_new_line(self, line, is_full_line):
+        if is_full_line:
```

Comment:
Does not work on real device when question to overwrite does not have newline char.

Reply:
Still not resolve problem. Sending y or n cannot be under is_full_line.